src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -402,7 +411,7 @@ public void GivenTheQuantityIsOverTheMaxLength()
         public void GivenFillsInTheCatalogueSolutionEditFormWithValidData()
         {
             var order = (Order)Context["CreatedOrder"];
-            var deliveryDate = order.CommencementDate.Value;
+            var deliveryDate = order.CommencementDate.Value.AddDays(7);
```

Comment:
any reason for this?

Reply:
I must've left it in by accident. Why was it in in the first place? Because 'All data are valid'  scenario was failing even for 'on-demand' pricing even though it shouldn't. There's possibility that this is a bug, I'll revert this change and once the story is completed, this can be re-visited.